_posts/course-materials/earth-analytics/week-8/in-class/2017-03-08-automation02-function-example-r.md

```diff
@@ -44,7 +46,28 @@ setwd("~/Documents/earth-analytics")
 
 # load spatial packages
 library(raster)
+## Loading required package: sp
```

Comment:
@mbjoseph are you talking about it not loading "quietly"? i started to add a general "hide messages" statement at the top of each lesson - but my guess is i didn't get to these lessons yet as I know i need a gentle redo of them to add the "overlay()" raster processing function!! i'll add these things to my todo's in the issues